upstream-community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -205,7 +206,700 @@ spec:
                         fieldPath: metadata.namespace
                   - name: TZ
                     value: UTC
-                      
+    
+    # discovery.yaml 
+    kind: Role
```

Comment:
This role, rolebinding, and serviceaccount can be added as a permission in the `permissions` section of the CSV